eZ/Publish/SPI/Persistence/Content/Location/Trash/TrashedItemDeleteResult.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * File containing the TrashedLocation class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Persistence\Content\Location\Trash;
+
+use eZ\Publish\SPI\Persistence\ValueObject;
+
+class TrashedItemDeleteResult extends ValueObject
+{
+    /** @var mixed */
+    public $trashItemId;
+
+    /** @var mixed */
+    public $contentId;
+
+    /** @var bool */
+    public $itemRemoved = true;
+
+    /** @var bool */
+    public $contentRemoved = false;
+}
```

Comment:
Same comment as in case of SPI's `TrashedItemDeleteResultList` - go for API. `$itemRemoved` seems unnecessary/not modified anyway.